docs/typescript/migration.md

```diff
@@ -0,0 +1,71 @@
+## Migrating to and from JavaScript
+
+One of the advantages of Deno is that it treats TypeScript and JavaScript pretty
+equally. This might mean that transitioning from JavaScript to TypeScript or
+even from JavaScript to TypeScript is something you want to accomplish. There
```

Comment:
This might mean that transitioning **from JavaScript to TypeScript** or even **from JavaScript to TypeScript** is something you want to accomplish
Duplication?